web3/pm.py

```diff
@@ -389,13 +389,13 @@ def set_registry(self, address: Union[Address, ChecksumAddress, ENS]) -> None:
             addr_lookup = self.web3.ens.address(str(address))
             if not addr_lookup:
                 raise NameNotFound(
-                    "No address found after ENS lookup for name: {0}.".format(address)
+                    f"No address found after ENS lookup for name: {address!r}."
```

Comment:
insisted on `repr` over `str`?

Reply:
Yeah, we use bytestrings and strings interchangeably, and mypy wants to make sure that's intentional.